src/server.cc

```diff
@@ -537,12 +537,6 @@ void Server::cron() {
       }
     }
 
-    // check every 30 minutes
```

Comment:
I agree, we should discard that

Reply:
I think it's better to remove this function in the future.